mcp-server/src/core/utils/agentllm-update-subtask-tool-saver.js

```
@@ -0,0 +1,154 @@
+		const allTasksData = readJSON(tasksJsonPath);
+		if (!allTasksData || !Array.isArray(allTasksData.tasks)) {
+			const errorMsg = `Invalid or missing tasks data in ${tasksJsonPath}.`;
+			logWrapper.error(`agentllmUpdateSubtaskSave: ${errorMsg}`);
+			return { success: false, error: errorMsg };
+		}
+
+		const [parentIdStr, subIdStr] = subtaskIdToUpdate.split('.');
+		const parentId = parseInt(parentIdStr, 10);
+		const subId = parseInt(subIdStr, 10);
+
+		if (isNaN(parentId) || isNaN(subId)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)